src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -77,11 +78,18 @@ public class SnowflakeSinkTask extends SinkTask {
   // check connect-distributed.properties file used to start kafka connect
   private final int rebalancingSleepTime = 370000;
 
-  private static final LoggerHandler LOGGER = new LoggerHandler(SnowflakeSinkTask.class.getName());
+  // the dynamic logger is intended to be attached per task instance. the instance id will be set
+  // during task start,
+  // however if it is not set, it falls back to the static logger
+  private static final LoggerHandler STATIC_LOGGER =
```

Comment:
this is only used for the 'getTopicToTableMap' method, which in turn is called in start and in Utils. It maybe better to move the method into Utils so we can remove this static logger